friendsofsymfony/rest-bundle/2.2/etc/packages/fos_rest.yaml

```diff
@@ -0,0 +1,27 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSRestBundle/index.html
+fos_rest:
+# Uncomment to activate features
+#
+#    param_fetcher_listener:  true
+#    allowed_methods_listener:  true
+#    routing_loader: true
+#    view:
+#        view_response_listener:  true
+#    exception:
+#        codes:
+#            App\Exception\MyException: 403
+#        messages:
+#            App\Exception\MyException: Forbidden area.
+#    format_listener:
+#        rules:
+#            - { path: ^/api, prefer_extension: true, fallback_format: json, priorities: [ json, html ] }
+#    body_listener:
```

Comment:
It seems it's enabled by default (https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/DependencyInjection/Configuration.php#L264) so it should be entirely removed... sorry one more change :stuck_out_tongue: